app/views/govuk_publishing_components/components/_step_by_step_nav_header.html.erb

```diff
@@ -18,13 +18,17 @@
   if tracking_id
     tracking_options ||= ({ dimension96: tracking_id }).to_json
   end
+
+  local_assigns[:margin_bottom] ||= 0
```

Comment:
is there a reason why you use local_assigns rather than `margin_bottom ||= 0`?

Reply:
No. Good point, that would probably work just fine.